src/main/java/org/reaktivity/nukleus/mqtt/internal/stream/MqttServerFactory.java

```diff
@@ -981,16 +979,15 @@ private void onDecodePublish(
 
             String info = "info";
 
-            final MqttDataExFW dataEx = mqttDataExRW
-                                            .wrap(dataExtBuffer, 0, dataExtBuffer.capacity())
-                                            .typeId(mqttTypeId)
-                                            .topic(topicName)
-                                            .expiryInterval(15)
-                                            .contentType("message")
-                                            .format(f -> f.set(MqttPayloadFormat.TEXT))
-                                            .responseTopic(topicName)
-                                            .correlationInfo(c -> c.bytes(b -> b.set(info.getBytes(UTF_8))))
-                                            .build();
+            final MqttDataExFW dataEx = mqttDataExRW.wrap(dataExtBuffer, 0, dataExtBuffer.capacity())
+                                                    .typeId(mqttTypeId)
+                                                    .topic(topicName)
+                                                    .expiryInterval(15)
+                                                    .contentType("message")
+                                                    .format(f -> f.set(MqttPayloadFormat.TEXT))
+                                                    .responseTopic(topicName)
+                                                    .correlationInfo(c -> c.bytes(b -> b.set(info.getBytes(UTF_8))))
+                                                    .build();
```

Comment:
Some of these hardcoded fields should be coming from `MqttPublishFW` frame instead, right?